cmake/dependencies/AMReX.cmake

```
@@ -92,6 +92,7 @@ macro(find_amrex)
         set(AMReX_PARTICLES ON CACHE INTERNAL "")
         set(AMReX_PROBINIT OFF CACHE INTERNAL "")
         set(AMReX_TINY_PROFILE ON CACHE BOOL "")
+        set(AMReX_LINEAR_SOLVER_INCFLO OFF CACHE INTERNAL "")
```

Explicit on might be best:
```suggestion
        set(AMReX_LINEAR_SOLVERS_EM ON CACHE INTERNAL "")
        set(AMReX_LINEAR_SOLVER_INCFLO OFF CACHE INTERNAL "")
```